### ISSUE TITLE
[WRECK] Issues with some wrecked items

### ISSUE THREAD

**Issue by Singularia** [Regalis11/Barotrauma#7709]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
1. Wrecked Suit Locker accepts **only** standard (Green) Diving Suits.
2. Icon of a suit in Wrecked Suit locker is a bit smaller compared to standard Suit Locker.
![image](https://user-images.githubusercontent.com/62324831/145607216-c720cdef-44dd-4520-945e-bfd553b31eac.png)

3. All Wrecked Doors and Ammo Loaders are missing recently added `CONDITION_OUT` pin.
![image](https://user-images.githubusercontent.com/62324831/145607580-fc134ab6-b5e8-47fc-ae0b-b6e90a0e9709.png)

4. Wrecked Cannons are missing `SET_LIGHT` pin
![image](https://user-images.githubusercontent.com/62324831/145625365-31c67326-ea28-4cd1-ab0e-9bf32b212396.png)

5. Wrecked batteries' UI is wider than their non-wrecked variants but battery slots themselves are smaller.
![image](https://user-images.githubusercontent.com/62324831/145608906-776c6ab0-bf59-4108-8e7b-03c8174d27aa.png)

**Version**
Unstable 0.15.19.0

**Comment by Singularia** [comment]
6. There is a similar issue with Wrecked Loaders as mentioned earlier in Item 5 and 2. Their UIs are sized differently compared to their not-wrecked counterparts.
7. All Wrecked Loaders (intentionally?) have their `Non Interactable` setting turned "ON" by default.
![image](https://user-images.githubusercontent.com/62324831/145659871-96ef3a07-d6ab-4997-a873-baf0dab2d3c8.png)

**Comment by Singularia** [comment]
9. Wrecked reactor has 6 slots for ANY items, while normal one has 4 slots for rods.
a. In mentioned earlier 6 slots of a Wrecked Reactor can be stored (intentionally?) almost any items, from handheld weaponry, ammo and clothing to rods, tools and other electronics.
![image](https://user-images.githubusercontent.com/62324831/145665006-ecf5c3de-e040-4f1f-a477-cfa9975f5cd9.png)

**Comment by Singularia** [comment]
10. UI of Wrecked Oxygen Tank Shelf has the same problem as described in Item 2, 5 and 6
![image](https://user-images.githubusercontent.com/62324831/145668949-ac4293a3-d7b3-482c-b88d-31627b0c6566.png)

**Comment by Singularia** [comment]
11. UI of a Wrecked Fire Extinguisher has a similar issue as mentioned in Item 10
![image](https://user-images.githubusercontent.com/62324831/145707447-bd5b4bf1-b3d7-4bd5-a706-a81701c60bbc.png)

**Comment by Singularia** [comment]
**Description**
I decided to add some additional issues that I found regarding wrecked items to this ticket instead of creating a new ticket.

_Listing will continue from the previous number_

**Version**
Tested on Unstable 0.16.2.0

12. Wrecked cabinets can't be linked to any installation, but other items like a fabricator for example can be linked to such wrecked cabinets.

12(a). All wrecked items lack the "Allowed links" category altogether.
![small steel cab wrk](https://user-images.githubusercontent.com/62324831/152642924-a2062c09-5179-4d13-b16e-6befefc1714b.png)

13. Wrecked versions of (Med)Fabricator, Deconstructor and both O2 Generators lack storage space unlike wrecked reactor

14. Wrecked Medicine Cabinet can't be mirrored by X-axis, just like its normal variant.

15. Wrecked Diving Suit locker doesn't have a "POWER_IN" pin, unlike its normal variant.
